structure_tests/Makefile

```diff
@@ -19,22 +19,38 @@ VERSION_BUILD ?= 0
 
 VERSION ?= v$(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 
+GOOS ?= $(shell go env GOOS)
+GOARCH = amd64
+PROJECT := runtimes-common/structure_tests
+RELEASE_BUCKET ?= gcp-container-tools/structure-test
+
+SUPPORTED_PLATFORMS := linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
```

Comment:
does this actually build with darwin and windows?

Reply:
it builds on linux/darwin, but there's an issue with windows currently. planning on fixing in a follow-up PR

Reply:
i would just remove the windows entry then, since cross build will fail right?